portfolio/src/main/webapp/script.js

```diff
@@ -109,22 +109,55 @@ $(document).ready(() => {
  * Adds comments to the page.
  */
 function addComments() {
-  fetch('/data?limit=2').then(response => response.json()).then((comments) => {
+  fetch('/data').then(response => response.json()).then((comments) => {
     const commentContainer = document.getElementById('comment-container');
     comments.forEach((comment) => {
-      commentContainer.append(createListElement(comment));
+      commentContainer.append(createComment(comment));
     });
   });
 }
 
 /** 
  * Creates an <li> element containing text. 
```

Comment:
this method does a lot more now, so it might be good to beef up this comment

Reply:
Thank you, I didn't think to update the descriptions of functions I had added to -- done!